resources/views/components/input/label.blade.php

```diff
@@ -0,0 +1,19 @@
+{{--
+It's called a label but it isn't? 🤯 Yup, as it's just for styling 🎨
+The benefit of this approach is that you don't have to deal with
+"for" attributes on labels and matching input names. Usage:
```

Comment:
My worry here is that the naming may by confusing leaving users to use this component incorrectly.
Have other names been considered?

Reply:
I think nobody will use a label without an input. We also moved the label to the input folder because you have to use it like this: `<x-rapidez::input.label>` instead of: `<x-rapidez::label>` see also: [#451](https://github.com/rapidez/core/pull/451/files#diff-38d7765c3a5fdea6a907b6d43c615d9d1e0a9bcadca894e86bfa6dc3d3e14d37)

Reply:
@indykoning I get your concern about the potential confusion with the naming. However, I prefer using label because it accurately describes the component’s function. This component will always be wrapped around an input, reinforcing its role as a label. I think that using this term helps maintain readability and prevents misuse, making the component’s purpose immediately recognizable.

Typically, a label sits above an input, which is why we call it a label. By wrapping the text inside the label, we simplify it a bit and eliminate the need to add a “for” attribute.